gooddata-pandas/requirements.txt

```diff
@@ -1,2 +1,2 @@
 pandas>=1.0.0,<2.0.0
-importlib-metadata >= 1.0 ; python_version >= "3.7"
+importlib-metadata >= 1.0 ; python_version >= "3.8"
```

Comment:
Remove dependency. Starting by py3.8, it is integral part of python distribution. See https://docs.python.org/3/library/importlib.metadata.html.
I think, we do not use any feature added by later versions of python >= 3.9 because we require 1.0 or newer. See compatibility matrix in https://pypi.org/project/importlib-metadata/.

Do not forget to apply on all places.